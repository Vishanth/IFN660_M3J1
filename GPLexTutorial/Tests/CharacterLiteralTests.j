'\u03a9'
'\uFFFF'
'\t'
'\\'
'\''
'\177'