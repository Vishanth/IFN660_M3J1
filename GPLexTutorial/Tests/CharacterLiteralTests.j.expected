CHARACTER LITERAL ('\u03a9')
CHARACTER LITERAL ('\uFFFF')
CHARACTER LITERAL ('\t')
CHARACTER LITERAL ('\\')
CHARACTER LITERAL ('\'')
CHARACTER LITERAL ('\177')
EOF